src/components/GameCards/GameCardDefault.vue

```diff
@@ -23,6 +23,12 @@
         @click.native="openDetails"
       />
 
+      <game-progress
+        v-if="progress && list.view !== 'covers'"
```

Comment:
I know you probably copy-pasted this but `list.view` will never equal 'covers' because this component is only used for the default view. This is check is probably leftover from when we used to have a single component for all list views.

```suggestion
        v-if="progress"
```

Reply:
Same in line 20:

```
v-if="showGameRatings && list.view !== 'covers'"
```

can be changed to:

```
v-if="showGameRatings"
```